specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/compute.json

```diff
@@ -8113,6 +8113,60 @@
           "final-state-via": "azure-async-operation"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/hostGroups/{hostGroupName}/hosts/{hostName}/restart": {
```

Comment:
per @pilor (ARM reviewer): 

This description indicates that this API should be async but it is not right now. Other restart POST APIs ARE async (VM restart for example)

Reply:
The [code in CRP is async](https://msazure.visualstudio.com/One/_git/Compute-CPlat-Core?path=/src/CRP/Dev/ComputeResourceProvider/Service.Azure/API/Controllers/DedicatedHosts/DedicatedHostsController.cs&version=GBmaster&line=156&lineEnd=162&lineStartColumn=9&lineEndColumn=10&lineStyle=plain&_a=contents) for this API. Why is this not async, I see no difference between this and VM restart? Is there something I need to add?

Reply:
For example, this is what [VM restart looks like](https://msazure.visualstudio.com/One/_git/Compute-CPlat-Core?path=/src/CRP/Dev/ComputeResourceProvider/Service/API/Controllers/VirtualMachinesControllerBase.cs&version=GBmaster&line=835&lineEnd=852&lineStartColumn=9&lineEndColumn=10&lineStyle=plain&_a=contents).